### PR TITLE
docs(OpenGov): Update governance section

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [ ] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [ ] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: docs/docs/networks/picasso/governance.md [hunk 1]
```diff
@@ -1,28 +1,73 @@
-# Governance
+# Picasso OpenGov
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:1]: not just picasso

File: docs/docs/networks/picasso/governance.md [hunk 2]
```diff
@@ -1,28 +1,73 @@
...
+
+## OpenGov Parameters
+
+The following two tables provide information about the voting and decision-making processes for the two different collectives and tracks (Whitelist Track and Root Track). They summarize the support thresholds required for various actions, the time it takes for those actions to pass when specific support percentages are met, and the confirmation and decision periods for each track.
+
+
+
+| Collective           | Support Threshold | Whitelist Time (Passing) | Root Time (Passing) |
+|-----------------------|-------------------|--------------------------|----------------------|
+| Technical Committee     | 10%               | 12 hours        | 5 days 14 hours      |
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:41]: assuming approval 100 percent referenda will pass after this time with this much support

File: docs/docs/networks/picasso/governance.md [hunk 3]
```diff
@@ -1,28 +1,73 @@
...
+|-----------------------|-------------------|--------------------------|
+| Whitelist        | 30 minutes        | 4 days                   |
+| Root             | 1 day             | 7 days                   |
+
+The following table summarizes the rules and requirements related to certain actions within the OpenGov system. It provides a concise reference for understanding the conditions and costs associated with various actions in the OpenGov process.
+
+
+| Action        | Origin & Threshold         | Submission Deposit | Decision Deposit (Pica)  |
+|---------------|----------------------------|--------------------|--------------------------|
+| Submit        | Anyone from any collective | 1 PICA             | Root Track: 500,000 Pica  |
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:58]: no submission depost for cancel

**Review by kkast — DISMISSED** [review]

File: docs/docs/networks/picasso/governance.md (hunk 1 above)
Comment (JafarAz) [reply docs/docs/networks/picasso/governance.md:1]: do you mean Composable aswell? that has another page

File: docs/docs/networks/composable/composable-council.md [hunk 4]
```diff
@@ -25,93 +23,33 @@ Composable Council members are as follows:
...
+|-----------------------|-------------------|--------------------------|
+| Whitelist Track       | 30 minutes        | 4 days                   |
+| Sudo Track            | 1 day             | 7 days                   |
 
-The voting power of a token holder is determined by two factors: 
-the amount of tokens and the length of time it is bonded or locked. 
-This refers to when users lock their tokens for an extended period in order to gain increased voting power such that:
 
-    Voting power = LAYR tokens * vote multiplier
+Any member from the two collectives is allowed to Submit an origin. The Cancel origin requires 1/3 approval from the Technical Commitee and the Kill origin requires 1/2 of the Council approval to execute.
```
[the remark this answers is not in the document]
Comment (kkast) [reply docs/docs/networks/composable/composable-council.md:51]: the cancel of of referenda and kill referenda

File: docs/docs/networks/picasso/governance.md [hunk 5]
```diff
@@ -1,28 +1,74 @@
...
+## OpenGov Parameters
+
+The following two tables provide information about the voting and decision-making processes for the two collectives. They summarize the support thresholds required for various actions, the time it takes for those actions to pass when specific support percentages are met, and the confirmation and decision periods for each track. With X % of support, referenda can pass after Y duration (times in the table) since the beginning of referenda if the approval rate is above the approval curve. The decision deposit for Root is 500,000 PICA whereas for Whitelist, its is 50,000 PICA. 
+
+
+| Support Threshold | Whitelist Time (Passing) | Root Time (Passing) |
+|-------------------|--------------------------|----------------------|
+| 10%               | 12 hours        | 5 days 14 hours      |
+| 20%               | 3 hours 20 minutes       | 4 days 4 hours       |
+| 30%               | 1 hour 15 minutes        | 2 days 18 hours      |
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:42]: remove last 3 rows

File: docs/docs/networks/picasso/governance.md [hunk 6]
```diff
@@ -1,28 +1,74 @@
...
+| Whitelist        | 30 minutes        | 4 days                   |
+| Root             | 1 day             | 7 days                   |
+
+The following table summarizes the rules and requirements related to certain actions within the OpenGov system. It provides a concise reference for understanding the conditions and costs associated with various actions in the OpenGov process.
+
+
+| Action        | Origin & Threshold         | Submission Deposit | 
+|---------------|----------------------------|--------------------|
+| Submit        | Anyone from any collective | 1 PICA             | 
+| Cancel        | 1/3 Tech Committee        | 1 PICA             |
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:59]: cancel and kill doesnt have submission deposit. only submission, remove this column and add a sentence about it

File: docs/docs/networks/picasso/governance.md [hunk 7]
```diff
@@ -1,28 +1,74 @@
...
+- Root
+- Whitelisted Caller (Fast Track)
+
+Additional tracks are also planned to be added during Phase 1 such as General Admin, Treasury and more.
+
+The governance model during Phase 1 will introduce four separate roles, each serving a unique purpose:
+
+- Proposal Creation: Enabling the ability to create proposals, fostering innovation and community-driven development.
+- Voting: Allowing GOV holders to vote on proposals, ensuring that decisions reflect the collective will of the community.
+- Canceling Proposals/Slashing: Implementing mechanisms to cancel proposals and penalize those who voted maliciously.
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:28]: no penalization of voters, penalized who submitted and deposited decision deposit

**Review by kkast — CHANGES_REQUESTED** [review]

File: docs/docs/networks/composable/composable-council.md [hunk 8]
```diff
@@ -25,93 +23,33 @@ Composable Council members are as follows:
...
-   a super majority vote from the Community to overrule the Council’s referendum.
-2. In the case of a simple majority decision by the Council, 
-   a simple majority vote from the Community will decide the outcome of the Council’s referendum.
+## Composable OpenGov Parameters
 
-### Canceling a referendum 
+The following tables outline the governance procedures related to the Governance tracks.
 
-Once a proposal has been submitted for referendum, the referendum cannot be canceled until it has entered a voting round, unless proposals are found and deemed malicious (for example, faults in code additions) by a 60% supermajority of the Composable Council or a unanimous decision of the Technical Committee, in which case the referendum shall be revoked. All tokens locked by supporters of the proposal would be burned.
+| Support Threshold | Whitelist Time (Passing) | Root Time (Passing) |
```
Comment (kkast) [inline docs/docs/networks/composable/composable-council.md:36]: whitelist track time and root track time maybe

File: docs/docs/networks/picasso/governance.md [hunk 9]
```diff
@@ -1,28 +1,74 @@
...
+| Root             | 1 day             | 7 days                   |
+
+The following table summarizes the rules and requirements related to certain actions within the OpenGov system. It provides a concise reference for understanding the conditions and costs associated with various actions in the OpenGov process.
+
+
+| Action        | Origin & Threshold         | Submission Deposit | 
+|---------------|----------------------------|--------------------|
+| Submit        | Anyone from any collective | 1 PICA             | 
+| Cancel        | 1/3 Tech Committee        | 1 PICA             | 
+| Kill          | 1/2 Council                | 1 PICA             |
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:60]: add action: whitelist call 1/3 tech or 1/3 council

File: docs/docs/networks/picasso/governance.md [hunk 10]
```diff
@@ -1,28 +1,72 @@
...
+![root-curve](../root-track.png)
+*Approval curve for the Root Track*
+
+
+| Track Type            | Confirm Period    | Decision Period (Voting) |
+|-----------------------|-------------------|--------------------------|
+| Whitelist        | 30 minutes        | 4 days                   |
+| Root             | 1 day             | 7 days                   |
+
+Any member from the two collectives is allowed to submit a referenda. The Cancel referenda requires 1/3 approval from the Technical Committee and the Kill referenda requires 1/2 of the Council approval to execute. These actions represent varying levels of decision-making authority, from open submission to stringent consensus requirements. The Whitelist Track requires either a 1/3 approval from the Tech Committee or 1/3 council. The decision deposit for Root is 500,000 PICA whereas for Whitelist, its is 50,000 PICA.
```
Comment (kkast) [inline docs/docs/networks/picasso/governance.md:58]: to cancel referenda and to kill referenda

**Review by kkast — CHANGES_REQUESTED** [review]

**Review by kkast — APPROVED** [review]

**Pull request merged**